source/administration/batch-framework-job-replicate.rst

```diff
@@ -96,6 +96,10 @@ Optionally, the YAML can also define flags to filter which objects replicate, se
 
    You can replicate from a remote MinIO deployment to the local deployment that runs the batch job.
 
+.. versionchanged:: MinIO RELEASE.2024-08-03T04-33-23Z
+
+   v2 of the Batch Job Replicate API allows you to list multiple prefixes on the source to replicate from.
```

Comment:
Are multiple prefixes the only thing different with `apiVersion : v2`? The page doesn't otherwise discuss that there are multiple versions, I'm not sure if `v2` is also new. 

The other reference I see to `apiVersion` is on the batch expiration page, where it's noted as required to be `v1`. Otherwise it's just an easily-overlooked line in YAML examples. Seems like it's time for a discussion about version 2 of this API?

Funny that what prompted me to dig into this was a sentence starting with a lowercase character. 😆

Reply:
The only current change with `v2` is the prefix thing. And that's for backwards compatibility as I understand it.

Anyway, I've added a bit more detail that should help address this point.